Cookbook/Technical-Documents/DebuggingPushNotifications.md

```diff
@@ -0,0 +1,37 @@
+# Debugging Push Notifications
+
+To sucessfully debug Push Notification on your device make sure that you have proper configurations set up as described in this document.
+
+- [PubNub](#pubnub)
+- [TLDR](#tldr)
+
+## PubNub
+
+[Pub Nub](https://admin.pubnub.com/) is a service we use to deliver notifications to APNS (apple Push Notifications Service) or FCM (Firebase Cloud Messaging). If you don't have access to Pub Nub yet ask other members of the team to add your work email to our account.
+
+Each application that supports push notifications has a separate configuration on PubNub. Depending on your access level you may have access to all the apps (admin) or to specific apps. All the apps are grouped under `steven.hamblin` account.
```

Comment:
It's worth mentioning here - who can give access to required apps

Reply:
AFAIK we don't have specific PubNub "owner" (or any other service owner) so I wouldn't put it under someone specific prerogative. It should be part of on boarding checklist though.